lib/utils/parseCategoryApps.js

```diff
@@ -18,8 +18,23 @@ function checkIfCategoryIsReallyPaid (selectedCollection, allCategories) {
   return !firstApps[0].free;
 }
 
-function getParsedCluster (categoryObject, collection) {
-  const collections = {
+function getClusterMappings (collection) {
```

Comment:
I guess I didn't mention when you first introduced this module, but it is solving a rather complex problem and I'm pretty sure it's something that will likely need to update frequently when google rearranges things in the page so we could use a good comment explaining what it does, what clusters are and how this may need to be edited to map what happens in the page.

Reply:
I add some comments and took time to rename some variables to make it more readable.